indra_db/reading/read_db.py

```diff
@@ -500,23 +489,75 @@ def dump_statements_to_pickle(self, pickle_file):
         """Dump the statements into a pickle file."""
         self.starts['dump_statements_pkl'] = datetime.utcnow()
         with open(pickle_file, 'wb') as f:
-            pickle.dump([sd.statement for sd in self.statement_outputs], f)
+            pickle.dump([sd.statement for sd in self.result_outputs], f)
         print("Statements pickled in %s." % pickle_file)
         self.stops['dump_readings_pkl'] = datetime.utcnow()
         return
```

Comment:
For consistency, we should probably change the naming here as well. I don't think the code needs to change materially though.